data/_ui/tooltips.txt

```diff
@@ -745,94 +745,94 @@ tip "Special:"
 
 tip "energy damage / second:"
-	`When hitting a target this weapon reduces the target's energy by this amount per second, or half this amount if the target's shields are up.`
+	`Reduces the target's energy by this amount per second. Only half as effective when the target's shields are up.`
 
 tip "ion damage / second:"
-	`Reduces the target's energy. Ionization slowly wears off over time.`
+	`Reduces the target's energy. Only half as effective when the target's shields are up. Ionization slowly wears off over time.`
 
 tip "scrambling damage / second:"
-	`Causes weapon failures. Scrambling slowly wears off over time.`
+	`Increases the target's chance of weapon failure. Only half as effective when the target's shields are up. Scrambling slowly wears off over time.`
 
 tip "slowing damage / second:"
-	`Reduces the target's turn rate, acceleration, and top speed. Slowing wears off over time.`
+	`Reduces the target's turn rate, acceleration, and top speed. Only half as effective when the target's shields are up. Slowing wears off over time.`
 
 tip "disruption damage / second:"
-	`Disrupts the target's shields, allowing weapon damage to "leak through" to the hull even if shields are up. Shield disruption wears off over time, and adding additional shield disruption becomes more difficult as damage leaks past shields instead of disrupting them.`
+	`Allows weapon damage to "leak through" to the hull even if shields are up. Shield disruption wears off over time, and adding additional shield disruption becomes more difficult as damage leaks past shields instead of disrupting them.`
 
 tip "discharge damage / second:"
-	`Amount of shield damage over time dealt by this weapon per second when firing.`
+	`Shield damage dealt over time by this weapon per second.`
 
 tip "corrosion damage / second:"
-	`Amount of hull damage over time dealt by this weapon per second when firing.`
+	`Hull damage dealt over time by this weapon per second.`
 
 tip "leak damage / second:"
-	`Amount of fuel damage over time dealt by this weapon per second when firing.`
+	`Fuel damage dealt over time by this weapon per second.`
 
 tip "burn damage / second:"
-	`Amount of heat damage over time dealt by this weapon per second when firing.`
+	`Heat damage dealt over time by this weapon per second.`
 
 tip "% shield damage / second:"
-	`Relative shield damage, converted to damage per second to allow easy comparisons between different weapons.`
+	`Shield damage dealt per second, as a percentage of the target's total shield strength.`
 
 tip "% hull damage / second:"
-	`Relative hull damage, converted to damage per second to allow easy comparisons between different weapons.`
+	`Hull damage dealt per second, as a percentage of the taget's total hull strength.`
 
 tip "% minable damage / second:"
-	`Relative minable damage, converted to damage per second to allow easy comparisons between different weapons.`
+	`Minable damage dealt per second, as a percentage of the target's total healt.`
```

Comment:
```suggestion
	`Minable damage dealt per second, as a percentage of the target's total health.`
```

Reply:
~~Don't we have a spellcheck CI or something?~~